### PR TITLE
chore: remove antd from correlation table

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Towards https://github.com/PostHog/posthog/issues/13624

## Changes

- Ported both tables over
- Remove some styles in favour of Tailwind classes

Plenty could be shared here but I just wanted to get the minimum done for antd removal.

## How did you test this code?

- Added a story

|Before|After|
|----|----|
| Screenshot 2024-03-11 at 17 53 50 | Screenshot 2024-03-12 at 10 40 19 |

Screenshot 2024-03-12 at 10 39 48
Screenshot 2024-03-12 at 10 39 38
Screenshot 2024-03-12 at 09 35 45
`(Empty string)` here is just because I didn't go to the effort of mocking the correlation endpoint twice for the nested properties and the parsing then doesn't work. Should be fine in production

### CODE REVIEW

File: frontend/src/mocks/fixtures/api/projects/team_id/insights/funnelCorrelation.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (daibhin) [inline frontend/src/mocks/fixtures/api/projects/team_id/insights/funnelCorrelation.json:1]: Random data I came up with based on an example response from production

File: frontend/src/scenes/insights/views/Funnels/CorrelationActionsCell.tsx [hunk 2]
```diff
@@ -82,8 +82,16 @@ const CorrelationActionsCellComponent = ({ buttons }: CorrelationActionsCellComp
...
             actionable
             onClickOutside={() => setPopoverOpen(false)}
-            overlay={buttons.map((props, index) => (
-                <LemonButton key={index} fullWidth {...props} />
+            overlay={buttons.map(({ onClick, ...props }, index) => (
+                <LemonButton
+                    key={index}
+                    fullWidth
+                    onClick={(e) => {
+                        setPopoverOpen(false)
```
Comment (daibhin) [inline frontend/src/scenes/insights/views/Funnels/CorrelationActionsCell.tsx:90]: Improvement over existing. Popover stayed open when you opened the modal 🤮

**Review by thmsobrmlr — CHANGES_REQUESTED** [review]
Thanks for cleaning this up!

You already saw the other PR that's not causing the merge conflicts here. We can revert that or remove the antd style adaptations from there once antd is fully gone.

Found a couple of issues here though:
1. The pagination for correlated events doesn't work. My local insight should have 4 pages with 5 items (see the 20 items before I click), but only 2 pages are displayed. Clicking on the second page causes this:
    
    

2. Uncollapsing another event should not trigger a reload of the previously opened ones:

    

3. (minor): The white styling here looks a bit off, since the rest is tan:

    Screenshot 2024-03-12 at 12 14 15

    Ideally we have a similar styling to before (tan header, white content) I guess. Not a blocker and I know that's subjective as well.

**Comment by daibhin** [comment]
@thmsobrmlr thanks for taking a look.

I reverted to using your empty state. It seems like Product Analytics use the `Empty` component in a bunch of places so I'm going to use them for now and we can replace them all at once when we decide on a good icon, etc.

Is there a way to generate correlation data locally? Part of the reason I added the story was because I struggled to test some of this locally

**Comment by daibhin** [comment]
@thmsobrmlr I certainly fixed point 1) in [this commit](https://github.com/PostHog/posthog/pull/20852/commits/6b04c8eed063dfbe4821b4944ef7f969f62be0fb). And hopefully 2) too 🤞

**Comment by thmsobrmlr** [comment]
> Is there a way to generate correlation data locally? Part of the reason I added the story was because I struggled to test some of this locally

Yeah it's definitely hard to create data/funnels that even have a correlation analysis. The easiest way is probably to use the "Activation" funnel from demo data.

**Comment by thmsobrmlr** [comment]
> @thmsobrmlr I certainly fixed point 1) in [this commit](https://github.com/PostHog/posthog/pull/20852/commits/6b04c8eed063dfbe4821b4944ef7f969f62be0fb). And hopefully 2) too 🤞

Yup, (1) looks good now. (2) is still present, but I just found out the issue is also present in the version in `master`. Good to ship this then :)

**Review by thmsobrmlr — APPROVED** [review]

**Pull request merged**